code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -171,8 +171,24 @@
 	new /obj/item/ammo_magazine/rocket/recoilless/light(src)
 	new /obj/item/storage/backpack/rpg(src)
 
+/obj/item/storage/box/tx29_system
+	name = "\improper TX-29 crate"
+	desc = "A large case containing a TX-29 squad support weapon and its grenade launcher payload. Standard rifle magazines sold separately. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
```

Comment:
Also remove this box, just put all the parts in the vendors seperately

Reply:
If people are hoarding ammo it wouldnt matter because the same person would just vend all the boxes and take the ammo, so do it

Reply:
Ok it's back in unga vendor without the crate.